app/models/snap_application.rb

```diff
@@ -54,6 +55,16 @@ class SnapApplication < ApplicationRecord
     where.not(id: Export.emailed_client.succeeded.application_ids)
   end)
 
+  def drive_status
+    if driver_applications.any? && driver_errors.empty?
+      :drive_in_flight
```

Comment:
given the conditionals I am reading above, I wouldn't expect this status to be `in flight` -- maybe something like `successfully completed` instead?  What do you think, is there a clearer way we could know that a drive is in progress?

Reply:
yeah good call!